web/front/src/styles/_base.sass

```diff
@@ -13,7 +13,7 @@
 //-- RESET + BASE --//
 html 
   background: url('../assets/magic-verse.png') no-repeat center center fixed
-  background-size: cover
+  background-size: cover !important
```

Comment:
You should never use !important in CSS for priority issues https://uxengineer.com/css-specificity-avoid-important-css/ 
Can you explain the necessity of important here ?